src/styles/main.scss

```diff
@@ -0,0 +1,62 @@
+/* #region Normalize */
+body {
+  margin: 0;
+}
+/* #endregion */
+
+/*#region Variables */
+$pin-size: 24px;
+/* #endregion */
+
+/* #region Mixins */
+
+@mixin position {
+  position: absolute;
+  left: 50%;
+  bottom: 50%;
+}
+
+@mixin pin_size($pin_size) {
```

Comment:
i think circle would be a more descriptive name of what it is

Reply:
Ok. Changed to circle.